src/hotspot/os/linux/cgroupV2Subsystem_linux.cpp

```diff
@@ -212,6 +234,20 @@ char* CgroupV2Subsystem::mem_limit_val() {
   return os::strdup(mem_limit_str);
 }
 
+// the kmem (kernel memory limits) are not separately accounted in cgroups v2
+jlong CgroupV2Subsystem::kernel_memory_usage_in_bytes() {
+  return OSCONTAINER_ERROR;
+}
+
+jlong CgroupV2Subsystem::kernel_memory_limit_in_bytes() {
+  return OSCONTAINER_ERROR;
+}
+
+jlong CgroupV2Subsystem::kernel_memory_max_usage_in_bytes() {
+  return OSCONTAINER_ERROR;
+}
+
+
```

Comment:
These should really not be needed if we make the `print_version_specific_info(...)` function implementation defined.